data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+			`	You're about to go forward, when you're stopped by Tugroob. They crawl around the entrance, looping it several times, without ever going further in, only sometimes stopping, and slowling reaching one leg inside, then back after apparently touching something.`
+			`	They come back down, and say, "Request support equipment, to get through safely, we must."`
+			`	Using their communication device, they call for support from Heliarch agents, warning them about the rockfall. Though you're well protected from the rockfall there, you hear several rocks fall rapidly, as Tugroob says the Heliarch are coming.`
+			`	Shortly after, several dozen Heliarch agents are there, some of them with certain cutting tools you can't quite identify, that they use to surgically cut down what Tugroob was slowly examining before, several extremely thin wires, connected to some type of machinery hidden within the wooden beams.`
+			`	After all is done, the agents storm the place, but find noone there. What they do find are dozens upon dozens of crates, filled to the brim with Heliarch staves and their blaster guns.`
+			`	They escort you two back outside, as the agents prepare to bring the smuggled weapons to their own ships. Though the storm is still raging on, you can make out the shape of a few Heliarch Interdictors parked nearby your own ship, right before you take off.`
+			`	You two regroup with Iplora and Kirrie, as they ask Tugroob for more details on the whole ordeal.`
+				accept
+			label anotherholeintheground
+			`	You two carefully make your way down the hole, down to the bottom, around ten meters below the flat area with the barn.`
+			`	As you were coming down, Tugroob pointed to the cracked, rocky walls, saying that most of them were unstable, and you should watch out for rockfall.`
+			`	Looking around, you two spot several smaller holes on the walls, many being just the right size for a human or Saryd arm to squeeze in, and a few larger cracks, that you two shouldn't have much trouble going through.`
+			choice
+				`	(Reach your arms inside the smaller holes to look for anything there.)`
+					goto sandyarm
+				`	(Go for the larger cracks and see if they lead to anything.)`
+					goto rockraider
+			label sandyarm
+			`	You carefully go about putting your arm inside a few of the holes, with Tugroob helping you up for the ones you couldn't reach, but ultimately you find nothing.`
+			`	Many small rocks fell as you went about the whole thing, but none hit neither of you, miraculously. Unfortunately, the storm above is bringing in too much sand now for you to even look around for more holes, or the exact location of the larger cracks you saw before.`
+			`	"To remain here for long, dangerous, will be, Captain," Tugroob says, as they urge you to cling to them. They carefully crawl back up, and you two take refuge inside the barn.`
```

Comment:
```suggestion
			`	"To remain here for long, dangerous, it will be, Captain," Tugroob says as they urge you to cling to them. They carefully crawl back up, and you two take refuge inside the barn.`
```